app/client/packages/design-system/ads/src/Templates/EntityExplorer/EntityListTree/EntityListTree.tsx

```
@@ -0,0 +1,74 @@
+  return (
+    <Flex flex="1" flexDirection="column">
+      {props.items.map((item) => (
+        <Flex flex="1" flexDirection="column" key={item.id}>
+          <EntityItemWrapper
+            alignItems="center"
+            data-depth={currentDepth}
+            data-disabled={item.isDisabled || false}
+            data-selected={item.isSelected}
+            flexDirection="row"
+          >
+            {item.children && item.children.length ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)